### ISSUE TITLE
hebocon pdx

### ISSUE THREAD

**Issue by max-mapper** [max-mapper/messages#41]
cc @nvcexploder @BenSJones

We are organizing a [hebocon](https://www.youtube.com/watch?v=46ivFpsmEVQ) for Portland.

TODO list for now:
- [ ] pick date for first organizers meetup
- [ ] find + invite organizers
- [x] squat twitter account (@heboconpdx)

TODO for first meeting (or after):
- [ ] pick date for sumo day
- [ ] pick venue for sumo day
- [ ] pick venue for robot building parties
- [ ] decide on exact format + logistics (in person at first meeting)
- [ ] create outreach team to do personal outreach to potential attendees
- [ ] add to calagator
- [ ] schedule robot building parties leading up to the sumo day
- [ ] create facebook event + add admins
- [ ] logo
- [ ] static site

also the hebocon organizers facebook requests we do this when ready:

```
At the time like this, please post to here.
* When you join this group.
- Write clearly where you would like to host Hebocon in. If there already are organizers in the same area, they can find you.
* When you decide the place and the date of your Hebocon.
- Please notice whether I can annouce it on our Facebook page or not. If I can, please add a short message from you to Hebocon fans around world.
* When your Hebocon is finished.
- If you can post the photos and videos here, we will introduce on our Facebook page! If you have too much datas to post, please send to me via email or something.
```

additionally, here is a rulebook: http://portal.nifty.com/hebocon/files/HEBOCON_simple_rule_book_v1.zip

**Comment by max-mapper** [comment]
I think https://ctrlh.org/ would be a good venue for and/or the parties or the competition. i'll try and get some photos and put em here

**Comment by nvcexploder** [comment]
looks like heboconpdx is already taken, also. Well done! :D

any ideas on other folks we can recruit?

**Comment by max-mapper** [comment]
@nvcexploder I just sent DMs/emails to a few people

**Comment by bensjones** [comment]
Here is the dropcam of ^h - https://ctrlh.org/dropcam/. I agree, it is a great meetup spot.

I have some contacts I think would be interested in being involved. I'll send out emails tonight.

Starting next week, Thursdays and Sundays are wide open for me.

**Comment by max-mapper** [comment]
no word back from the people I reached out to :(

**Comment by nvcexploder** [comment]
I will reach out to some folks too, then.

**Comment by max-mapper** [comment]
hey all, lets do a mini speed hebocon testrun at https://ctrlh.org/ on thursday march 5th (1 week from tomorrow). they will be having an open house that night, and some of the ctrlh (aka "^H") organizers suggested we make it into a hack night

I'm thinking we can invite any interested people to attend, everyone should bring some wacky craft supplies, and we can spend 2 hours making bots and then having a mini sumo battle, basically a super small scale test run of the event to give us more ideas on logistics.

there might be some other good potential organizers who show up to the open house as well

cc @kwugirl

**Comment by max-mapper** [comment]
also i'll be there tomorrow night if anyone wants to hang out in general http://calagator.org/events/1250467958

**Comment by nvcexploder** [comment]
Yes. I'm in! :)

**Comment by bensjones** [comment]
Yes and Yes. I'm way behind on my communications for this. Jon from ^h mentioned it a few weeks ago and I thought it was a great idea. Sounds like you talked with him and got it up here. Yay!

I'll be there tonight and would love to talk more. See ya tonight.

**Comment by bensjones** [comment]
Also, I have 1/4" plywood for the platform, so we have the ring for next week.

**Comment by max-mapper** [comment]
@bensjones hey sorry can't make it tonight actually, I came down with some bad allergies or potentially a cold (can't really tell) but i'm gonna stay in bed today

**Comment by bensjones** [comment]
Cool. @nvcexploder are you going to make it to ctrlh tonight?

**Comment by nvcexploder** [comment]
Gah! No - I can't. I can next week though!

**Comment by shama** [comment]
:+1: My son and I will be there March 5th, thanks for inviting us @maxogden! I'll try and scrounge up some craft supplies to bring.

**Comment by max-mapper** [comment]
see ya'll at 6! http://calagator.org/events/1250468002

**Comment by bensjones** [comment]
That was pretty fun. I do want to make some subtle adjustments and have a few more people helping on the scoring and bracket system, but I think this could work. Also, age groups are a must for the main event later this year.

What did you all think?

**Comment by shama** [comment]
We had a blast! Thanks for hosting and having us!

**Comment by ungoldman** [comment]
:+1: this happened

**Comment by max-mapper** [comment]
we should schedule the next one! how about same day as
http://nodeschool.io/international-day/ and then we can do both events at
CTRL+H

**Comment by ungoldman** [comment]
@maxogden +1 if <kbd>CTRL</kbd><kbd>H</kbd>ers are cool with it!

**Comment by bensjones** [comment]
This could work. Let me check with Jon about the availability.

**Comment by nvcexploder** [comment]
Yaassssssssss.

**Comment by max-mapper** [comment]
@bensjones how about Saturday, May 23rd from 12 - 6PM at CTRL+H? lemme know if that works an I'll put it on calagator and open a separate issue with event info to link to from calagator

**Comment by ungoldman** [comment]
@maxogden would be good to get it on http://nodeschool.io/globe/ & http://www.meetup.com/pdxnode/ too, no?

**Comment by bensjones** [comment]
@maxogden This time is open at ctrlh. We can reserve both meeting spaces (main meeting space and the craft lab) if needed. Thoughts on a meetup description? Let me know and I'll add it to the ctrlh's meetup events.

**Comment by max-mapper** [comment]
@ngoldman ya!

description:

## Hebocon (crappy robot contest) and NodeSchool International Day extravaganza!

On Saturday, May 23rd from 12PM - 5PM we will be having a coding + bot battling party at CTRL+H.

There will be an afternoon of learning how to code AND/OR building a Hebocon bot.

Heard of Hebocon? It is a sumo style battle competition for crappy robots! Rules are here http://portal.nifty.com/hebocon/files/HEBOCON_simple_rule_book_v1.pdf Come hang out, build a crappy robot that battles against other crappy robots. Bring an old RC car or other electronics to assemble a robot for the competition.

May 23rd is [NodeSchool International Day](http://nodeschool.io/international-day/), so in addition to the Hebocon you can also show up with a laptop and learn some JavaScript. There will be JavaScript mentors available. If there is enough interest we can do the NodeSchool NodeBots workshop: https://github.com/tableflip/nodebot-workshop

**Family friendly!** Kids encouraged. There will be a separate kids sumo bracket for kid-made bots.

#### Schedule
- 12pm - Show up!
- 12pm - 4pm build bots and/or do nodeschooling
- 4pm - 5pm - Hebocon battle!

**Comment by ungoldman** [comment]
:sparkles: http://www.meetup.com/pdxnode/events/222514015/ :sparkles: 

thx @chrisdickinson

**Comment by ungoldman** [comment]
:sparkles: http://calagator.org/events/1250468498 :sparkles: 

thx @onewheelskyward

**Comment by ungoldman** [comment]
I don't have http://nodeschool.io/globe/ powers, is that a thing that can be PRed? I imagine so

**Comment by max-mapper** [comment]
@ngoldman on it

**Comment by bensjones** [comment]
:sparkles: http://www.meetup.com/CTRL-H/events/222517630/ :sparkles:

**Comment by ungoldman** [comment]
✅  this happened
